yaksh/models.py

```diff
@@ -648,6 +648,9 @@ def _create_unit_copy(self, user):
                 order=self.order, type="lesson", lesson=new_lesson)
         return new_unit
 
+    def __str__(self):
+        return self.lesson.name if self.lesson else self.quiz.description
```

Comment:
Though not a problem, but in such method it is always better to avoid Foreign key null true. Lesson/Quiz can be null. Or else the case should be handled.
Just a suggestion.

Reply:
I changed the condition to check for the `type` field in the learning unit model instead of directly checking the foreign key.

Reply:
Looks good👍thanks